components/ProductCardEarn.tsx

```diff
@@ -16,7 +59,12 @@ export function ProductCardEarn({ cardData }: ProductCardEarnProps) {
 
   const maxMultiple = one.div(cardData.liquidationRatio.minus(one))
   const tagKey = productCardsConfig.earn.tags[cardData.ilk]
+
   const sevenDayAverage = new BigNumber(0.1201) // TODO to be replaced with calculations
+  const unprofitable = false // TODO to be replaced with calculations
+
+  const rightSlotValue = formatPercent(cardData.stabilityFee.times(100), { precision: 2 })
+  const leftSlotValue = formatPercent(sevenDayAverage.times(100), { precision: 2 })
```

Comment:
Could be good to call these something else?

```suggestion
  const stabilityFeePercentage = formatPercent(cardData.stabilityFee.times(100), { precision: 2 })
  const yieldAsPercentage = formatPercent(sevenDayAverage.times(100), { precision: 2 })
```